<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1252</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0" columnstretch="0">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0">
      <item row="0" column="1">
       <widget class="QListWidget" name="StoriesList">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>ContextLine</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Content</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="GenerateButton">
            <property name="text">
             <string>Generate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="FlushButton">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Flush</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="2">
         <widget class="CodeEditor" name="StoryTextEdit"/>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="StoryNameEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Story Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Interpretations</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="InterpretationsEdit"/>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="SaveStoryButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="ContextLineLabel">
          <property name="maximumSize">
           <size>
            <width>1666</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <widget class="InterpretationsList" name="IntepretationsList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>124</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1252</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionOpen_Stories_Directory"/>
    <addaction name="actionOpen_Vsuvky_Directory"/>
    <addaction name="actionSave_Story"/>
    <addaction name="actionFlush_Stories_to_HDD"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuStyle">
     <property name="title">
      <string>Style</string>
     </property>
     <addaction name="actionWhite"/>
     <addaction name="actionDark_attempt"/>
    </widget>
    <addaction name="menuStyle"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionLicense"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionWhite">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>White</string>
   </property>
  </action>
  <action name="actionDark_attempt">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark attempt</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionOpen_Stories_Directory">
   <property name="text">
    <string>Open Stories Directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpen_Vsuvky_Directory">
   <property name="text">
    <string>Open Vsuvky Directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionSave_Story">
   <property name="text">
    <string>Save Story to memory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFlush_Stories_to_HDD">
   <property name="text">
    <string>Flush Stories to HDD</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>codeEditor.h</header>
  </customwidget>
  <customwidget>
   <class>InterpretationsList</class>
   <extends>QListWidget</extends>
   <header>interpretationslist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
